cmd/orphan-finder/main.go

```diff
@@ -107,8 +114,13 @@ func parseLogLine(sa certificateStorage, logger blog.Logger, line string) (found
 		return true, false
 	}
 	// OCSP-Updater will do the first response generation for this cert so pass an
-	// empty OCSP response
-	_, err = sa.AddCertificate(ctx, der, int64(regID), nil)
+	// empty OCSP response. We use `cert.NotBefore` as the issued date to avoid
+	// the SA tagging this certificate with an issued date of the current time
+	// when we know it was an orphan issued in the past. Because certificates are
+	// backdated we need to add the backdate duration to find the true issued
+	// time.
+	issuedDate := cert.NotBefore.Add(1 * backdateDuration)
```

Comment:
I think you can remove the `1 *`

Reply:
Haha. Yes..... Good catch!

I had copied the negative `-1 * backdateDuration` from the CA and didn't put much thought into the sensibility of `1 * backdateDuration` :rofl:

Reply:
Fixed in cff973f